internal/manager/telemetry/reports.go

```diff
@@ -78,11 +78,17 @@ func SetupAnonymousReports(
 	if !ok {
 		return nil, fmt.Errorf("malformed database configuration found in Kong client root")
 	}
+	routerFlavor, ok := cfg["router_flavor"].(string)
+	if !ok {
+		// version to old to use router flavor, de facto traditional
+		routerFlavor = "traditional"
+	}
 
 	fixedPayload := Payload{
 		"v":  metadata.Release,
 		"kv": kongVersion,
 		"db": kongDB,
+		"r":  routerFlavor,
```

Comment:
Apart from the broken tests I believe we could use perhaps `rf` for router flavor? Simply `r` might become useful at some point for something else

Reply:
:shrug: none of them are really useful beyond just being a constant string for the sorta-CSV k=v ingest.